### PR TITLE
[QA Prep Required][Migration Required] Feature/Update AsPredicted form to the latest version [ENG-141]

### PR DESCRIPTION
**Pull request by pattisdr**
## Purpose

The OSF offers the AsPredicted registration form as an option when creating a registration. 
AsPredicted has updated their registration form to be 9 questions, instead of 8 questions long. They have also modified the wording of some of the existing questions. We should remain up-to-date with registration forms.

## Changes
- Updated as-predicted schema to latest language with new questions
- Added new schema version to the db
- Modified how returning the latest version of a schema works - instead of just assuming version 2 is the latest version (this was hardcoded), return the latest of each name. (assuming schema names remain consistent between versions!)

## QA Notes

- Make some registrations ahead of time using AsPredicted schema.
- Verify afterwards that these registrations with old schema versions are still accessible (showing only 8 questions) and show up in search 
- Verify new registrations made with AsPredicted schema have 9 questions and likewise show up n search
- /v2/schemas/registrations/ shows latest AsPredictedSchema version 3. 

## Ticket

https://openscience.atlassian.net/browse/ENG-141

### CODE REVIEW

File: api_tests/nodes/views/test_node_draft_registration_detail.py [hunk 1]
```diff
@@ -14,7 +14,8 @@
 )
 from rest_framework import exceptions
 from test_node_draft_registration_list import DraftRegistrationTestCase
-from website.project.metadata.schemas import LATEST_SCHEMA_VERSION
+
+SCHEMA_VERSION = 2
```
Comment (pattisdr) [inline api_tests/nodes/views/test_node_draft_registration_detail.py]: "LATEST_SCHEMA_VERSION" isn't a special thing anymore.  As the versions of the schemas below change, this variable might need to change. Hardcoding in the test file instead.

File: osf/models/metaschema.py [hunk 2]
```diff
@@ -11,6 +11,31 @@
...
+    def get_latest_versions(self, only_active=True):
+        """
+        Returns a queryset of the latest version of each schema
+        :param str only_active: Only return active schemas
+        :return queryset
+        """
+        latest_schemas = self.filter(visible=True)
+        if only_active:
+            latest_schemas = latest_schemas.filter(active=True)
+        return latest_schemas.order_by('name', '-schema_version').distinct('name')
```
Comment (pattisdr) [inline osf/models/metaschema.py]: Assumes that schema's names remain consistent between versions.  If that assumption changes, we would need to handle this differently.    This groups schemas by name and selects the schema with the most recent version of that group.

Comment (brianjgeiger) [reply osf/models/metaschema.py]: https://openscience.atlassian.net/browse/REQS-58

Comment (pattisdr) [reply osf/models/metaschema.py]: yay for requirements!

**Pull request merged**